### PR TITLE
feat: Add hook for ingest stats

### PR DESCRIPTION
**Pull request by Macbeth98**
# Description
- Adds hook to get cluster Ingest stats from all nodes

# Checklist:

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: pkg/health/clusterStatsHandler.go [hunk 1]
```diff
@@ -211,6 +211,12 @@ func ProcessClusterIngestStatsHandler(ctx *fasthttp.RequestCtx, orgId uint64) {
 
 	pastXhours, granularity := parseIngestionStatsRequest(readJSON)
 	rStats, _ := usageStats.GetUsageStats(pastXhours, granularity, orgId)
+
+	if hook := hooks.GlobalHooks.AddMultinodeIngestStats; hook != nil {
+		log.Infof("Mani: Processing ClusterIngestStatsHandler: calling AddMultinodeIngestStats hook")
```
Comment (AndrewHess) [inline pkg/health/clusterStatsHandler.go:216]: Delete this log

File: pkg/health/clusterStatsHandler.go [hunk 2]
```diff
@@ -211,6 +211,12 @@ func ProcessClusterIngestStatsHandler(ctx *fasthttp.RequestCtx, orgId uint64) {
...
+		hook(rStats, pastXhours, uint8(granularity), orgId)
+	}
```
[the remark this answers is not in the document]
Comment (nkunal) [reply pkg/health/clusterStatsHandler.go:218]: remove that log line or remove your name

File: pkg/hooks/hooks.go [hunk 3]
```diff
@@ -58,6 +58,8 @@ type Hooks struct {
 		queryCount *uint64, totalResponseTime *float64)
 
 	AddMultinodeSystemInfoHook func(ctx *fasthttp.RequestCtx)
+	// rStats is of type usageStats.ReadStats
+	AddMultinodeIngestStats func(rStats interface{}, pastXhours uint64, granularity uint8, orgId uint64)
```
Comment (nkunal) [inline pkg/hooks/hooks.go:62]: we should follow the naming convention according to the rest of the funcs

File: pkg/health/clusterStatsHandler.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndrewHess) [reply pkg/health/clusterStatsHandler.go:218]: Good point from Lindy, unless you decided you don't need to return an error

[the remark this answers is not in the document]
Comment (Macbeth98) [reply pkg/health/clusterStatsHandler.go:217]: Yes I am not returning any error for this function.

**Review by nkunal — APPROVED** [review]

**Review by nkunal — APPROVED** [review]
lgtm

**Pull request merged**